db/migrate/20150312144232_migrate_websockets_setting.rb

```diff
@@ -0,0 +1,14 @@
+class MigrateWebsocketsSetting < ActiveRecord::Migration
+  def up
+    return unless encrypt = Setting.find_by_name("websockets_encrypt")
+    encrypt.settings_type = "boolean"
+    encrypt.value = (encrypt.value == true || encrypt.value == "true" || encrypt.value == "auto") ? true : false
```

Comment:
When the key + cert are empty, while websockets_encrypt is "auto", this changes the new value to true.  Should it check if the key + cert are defined and if not, set it to false?

I was surprised to see my auto/unset/unset config change to true/unset/unset when it's really false.